charts/apiclarity/values.yaml

```diff
@@ -39,6 +39,14 @@ global:
 
 apiclarity:
 
+  # Send notification of new API Findings to this backend
+  # notification_url_prefix: http://example.local:8888/
```

Comment:
Please use camel case `notificationUrlPrefix`
Also, `Send notification of new API Findings to this backend` - it is not only for `new API Findings`, right?